src/lib.rs

```diff
@@ -766,6 +889,30 @@ pub unsafe trait FromZeroes {
     }
 
     /// Creates an instance of `Self` from zeroed bytes.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # mod derive { pub use zerocopy_derive::FromZeroes; }
+    /// # use zerocopy::FromZeroes;
+    /// # use derive::*;
+    /// #
+    /// #[derive(FromZeroes)]
+    /// #[repr(C)]
+    /// struct PacketHeader {
+    ///     src_port: [u8; 2],
+    ///     dst_port: [u8; 2],
+    ///     length: [u8; 2],
+    ///     checksum: [u8; 2],
+    /// }
+    ///
+    /// let header: PacketHeader = FromZeroes::new_zeroed();
+    ///
+    /// assert_eq!([0, 0], header.src_port);
+    /// assert_eq!([0, 0], header.dst_port);
+    /// assert_eq!([0, 0], header.length);
+    /// assert_eq!([0, 0], header.checksum);
```

Comment:
To be consistent with [stdlib examples](https://doc.rust-lang.org/stable/std/ptr/struct.NonNull.html#examples-3), swap the order here:

```suggestion
    /// assert_eq!(header.src_port, [0, 0]);
    /// assert_eq!(header.dst_port, [0, 0]);
    /// assert_eq!(header.length, [0, 0]);
    /// assert_eq!(header.checksum, [0, 0]);
```